storage/core/src/main/java/com/hortonworks/registries/storage/impl/jdbc/JdbcStorageManager.java

```diff
@@ -245,4 +243,18 @@ public void init(Map<String, Object> properties) {
         this.queryExecutor.setStorableFactory(storableFactory);
     }
 
+    @Override
+    public void beginTransaction(TransactionIsolation transactionIsolationLevel) throws StorageException {
```

Comment:
These throw `StorageException` but `TransactionManager`'s methods doe snot throw any exceptions. We may not want to declare to throw any exception here. Same for rollback/commit methods.

Reply:
Thanks Satish, I removed storage exception for QueryExecutor methods, looks like I missed for StorageManager methods. I will remove this.